blazar_dashboard/content/leases/templates/leases/calendar.html

```diff
@@ -3,40 +3,39 @@
 {% block title %}{% trans "Leases" %}{% endblock %}
 
 {% block page_header %}
-  {% include "horizon/common/_page_header.html" with title=_("Host Calendar") %}
+{% include "horizon/common/_page_header.html" with title=calendar_title %}
 {% endblock page_header %}
 
 {% block main %}
-  <form class="form-inline" name="blazar-gantt-controls">
-    <div class="form-group gantt-group">
-      <button class="btn btn-sm btn-default gantt-quickdays" data-gantt-days="1">1d</button>
-      <button class="btn btn-sm btn-default gantt-quickdays" data-gantt-days="7">1w</button>
-      <button class="btn btn-sm btn-default gantt-quickdays" data-gantt-days="31">1m</button>
+  <form class="form-inline" name="blazar-calendar-controls">
+    <div class="form-group calendar-group">
+      <button class="btn btn-sm btn-default calendar-quickdays" data-calendar-days="1">1 {% trans "day" %}</button>
+      <button class="btn btn-sm btn-default calendar-quickdays" data-calendar-days="7">7 {% trans "days" %}</button>
+      <button class="btn btn-sm btn-default calendar-quickdays" data-calendar-days="31">31 {% trans "days" %}</button>
```

Comment:
Do you think it makes more sense to make this 30 days? It's more of a round number is all. I know it was at 31 before.